packages/p2p-v2/src/pages/buy-sell/components/SortDropdown/SortDropdown.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { TSortByValues } from '@/utils';
+import { LabelPairedChevronDownMdRegularIcon } from '@deriv/quill-icons';
+import { Button, Dropdown, useDevice } from '@deriv-com/ui';
+import SortIcon from '../../../../public/ic-cashier-sort.svg';
+
+type TSortDropdownProps = {
+    list: { text: string; value: string }[];
+    onSelect: (value: TSortByValues) => void;
+    setIsFilterModalOpen: (value: boolean) => void;
+    value: TSortByValues;
+};
+
+const SortDropdown = ({ list, onSelect, setIsFilterModalOpen, value }: TSortDropdownProps) => {
+    const { isMobile } = useDevice();
+
+    if (isMobile) {
+        return (
+            <Button
+                className='w-[3.2rem]'
+                icon={<SortIcon className='absolute' data-testid='dt_p2p_v2_sort_dropdown_button' />}
+                onClick={() => setIsFilterModalOpen(true)}
+                variant='outlined'
+            />
+        );
+    }
+
+    return (
+        <Dropdown
+            dropdownIcon={<LabelPairedChevronDownMdRegularIcon />}
+            label='Sort by'
+            list={list}
+            name='hi'
```

Comment:
name=hi? 🙄

Reply:
oh god i forgot to remove this 😆 thanks for pointing this out